src/base/bittorrent/session.cpp

```diff
@@ -2412,13 +2413,34 @@ void Session::networkConfigurationChange(const QNetworkConfiguration& cfg)
     }
 }
 
+bool Session::addressPatternMatches(const QString& ipAddress, const QString& pattern)
+{
+    QStringList ipPieces = ipAddress.split(".");
+    QStringList patternPieces = pattern.split(".");
+
+    if (ipPieces.size() != patternPieces.size())
+    {
```

Comment:
Opening bracket in the same line with if's: `if () {`
The same for `else`, `else if ()` and `for ()`.